tests/test_request_handler.py

```diff
@@ -0,0 +1,162 @@
+                response=json.dumps({'key': 'value'}))
+        with self.assertRaises(APIErrorException) as context:
+            self.rq._get(self.dummy_url)
+        self.assertTrue("This resource does not exist. "
+                        "Check parameters" in context.exception)
+
+    @mock.patch('requests.get')
+    def test_too_many_requests_code(self, mock_call):
+        mock_call.return_value = mock.MagicMock(
+                status_code=requests.codes.too_many_requests,
+                response=json.dumps({'key': 'value'}))
+        with self.assertRaises(APIErrorException) as context:
+            self.rq._get(self.dummy_url)
+        self.assertTrue("You have exceeded your allowed "
+                        "requests per minute/day" in context.exception)
+
+    @mock.patch('test_request_handler.Stdout.live_scores')
+    @mock.patch('requests.get')
+    def test_get_live_scores_ok(self, mock_request_call, mock_writer):
+        mock_request_call.side_effect = [mocked_requests_get({'games': [1, 2]}, 200)]
+        mock_writer.return_value = mock.Mock()
+        self.rq.get_live_scores(True)
+        mock_writer.assert_called_once()
+
+    @mock.patch('test_request_handler.click.secho')
+    @mock.patch('test_request_handler.Stdout.live_scores')
+    @mock.patch('requests.get')
+    def test_get_live_scores_0_games(self,
+                                     mock_request_call, mock_writer,
+                                     mock_click):
+        mock_request_call.side_effect = [mocked_requests_get({'games': []}, 200)]
+        mock_writer.return_value = mock.Mock()
+        self.rq.get_live_scores(True)
+        mock_click.assert_called_with("No live action "
+                                      "currently", fg="red", bold=True)
+
+    @mock.patch('test_request_handler.click.secho')
+    @mock.patch('test_request_handler.Stdout.live_scores')
+    @mock.patch('requests.get')
+    def test_get_live_scores_error(self,
+                                   mock_request_call, mock_writer,
+                                   mock_click):
+        mock_request_call.side_effect = [mocked_requests_get({'games': [1, 2]}, 400)]
+        mock_writer.return_value = mock.Mock()
+        self.rq.get_live_scores(True)
+        mock_click.assert_called_with("There was problem getting "
+                                      "live scores", fg="red", bold=True)
+
+    @mock.patch('test_request_handler.Stdout.team_scores')
+    @mock.patch('requests.get')
+    def test_get_team_scores_ok(self,
+                                mock_request_call, mock_writer):
+        mock_request_call.side_effect = [mocked_requests_get({'fixtures': [1, 2]}, 200)]
+        mock_writer.return_value = mock.Mock()
+        self.rq.get_team_scores("AFC", 6, True, True)
+        mock_writer.assert_called_once()
+
+    @mock.patch('test_request_handler.click.secho')
+    @mock.patch('test_request_handler.Stdout.team_scores')
```

Comment:
Is there a reason this cannot be `@mock.patch('soccer.writer.Stdout.team_scores')` and `@mock.patch('click.secho')` etc?